.pre-commit-config.yaml

```diff
@@ -56,3 +56,11 @@ repos:
         language: golang
         require_serial: true
         files: ^kubectl-plugin/
+
+  - repo: local
+    hooks:
+      - id: validate-helm-charts
+        name: validate helm charts with kubeconform
+        entry: bash -c 'version="0.6.7"; [ "$(kubeconform --v | grep -oP "(?<=v)[\d\.]+")" = "$version" ] || echo "kubeconform version is not $version"; bash scripts/validate-helm.sh'
+        language: system
+        pass_filenames: false
```

Comment:
This is wrong. You must separate them to 2 hooks. Otherwise it won't catch the error.

See the following image, I set the version to "invalid" but it still passed.
![image](https://github.com/user-attachments/assets/7aa5fbde-92f2-4003-9aee-943fa59111a2)